README.md

```diff
@@ -150,6 +150,8 @@ This is example walks through using a Triton Server and Python client using MONA
 An example of experiment management with [Aim](https://aimstack.io/aim-monai-tutorial/), using 3D spleen segmentation as an example.
 ##### [MLFlow](./experiment_management/spleen_segmentation_mlflow.ipynb)
 An example of experiment management with [MLFlow](https://www.mlflow.org/docs/latest/tracking.html), using 3D spleen segmentation as an example.
+#### [MLFlow with MONAI workflow](./experiment_management/workflow_integrate_mlflow.ipynb)
```

Comment:
I would suggest to change the name to "bundle_integrate_mlflow" and update all the descriptions accordingly.

Thanks.